### ISSUE TITLE
example_scenes.py #3 gives "Latex error converting to dvi"

### ISSUE THREAD

**Issue by tj-murphy** [3b1b/manim#1212]
Hi!

Today I installed manim on Ubuntu 20.04, using `pip3 install manimlib`.

I then tried running the example by using `python3 -m manim example_scenes.py -pl`. This gave me a Module Not Found for manim. Once I took out the `python3 -m` and called manim directly, I got a file not found for example_scenes.py, so I went to this repo and downloaded that file individually.

I was then able to run the example script, and it prompts me for which animations to render. #1 and #2 work great, but once I get to the third, I get the following output:
```
Choice(s): 3
sh: 1: latex: not found



Traceback (most recent call last):
  File "/home/human/.local/lib/python3.8/site-packages/manimlib/extract_scene.py", line 155, in main
    scene = SceneClass(**scene_kwargs)
  File "/home/human/.local/lib/python3.8/site-packages/manimlib/scene/scene.py", line 75, in __init__
    self.construct()
  File "example_scenes.py", line 118, in construct
    decimal = DecimalNumber(
  File "/home/human/.local/lib/python3.8/site-packages/manimlib/mobject/numbers.py", line 36, in __init__
    self.add(*[
  File "/home/human/.local/lib/python3.8/site-packages/manimlib/mobject/numbers.py", line 37, in <listcomp>
    SingleStringTexMobject(char, **kwargs)
  File "/home/human/.local/lib/python3.8/site-packages/manimlib/mobject/svg/tex_mobject.py", line 42, in __init__
    file_name = tex_to_svg_file(
  File "/home/human/.local/lib/python3.8/site-packages/manimlib/utils/tex_file_writing.py", line 21, in tex_to_svg_file
    dvi_file = tex_to_dvi(tex_file)
  File "/home/human/.local/lib/python3.8/site-packages/manimlib/utils/tex_file_writing.py", line 69, in tex_to_dvi
    raise Exception(
Exception: Latex error converting to dvi. See log output above or the log file: media/Tex/701bbfb7cb8d210c.log
```
Any help would be appreciated; happy to post that log if it would be helpful but I don't want to be spewing documents if there's no need. Thanks!

Edit: Actually, now that I look, the log file doesn't exist. Looks like it wasn't actually produced. My Tex folder just contains the .tex files that it was attempting to render.

**Comment by tj-murphy** [comment]
Update: Turns out I didn't have Latex installed on my system. I installed it by using:

`sudo apt install texlive-latex-extra`

Then, I still ran into an error where it was missing "dsfont.sty". I gather that sty files are Latex add-on packages. I obtained the font sty by running

`sudo apt install texlive-fonts-extra`

Then I got yet another error, that I was missing "physics.sty". This one I fixed by:

`sudo apt-get install texlive-science`

After that, I was able to successfully run the examples.
